### DISCUSSION TITLE
Is domain wide delegation needed to access workspace groups?

### DISCUSSION THREAD

**Discussion by cpl-cpi** [dexidp/dex#2727] [Q&A]
Hi!
Actually we are wondering if the domain wide delegation is really needed to get access to google workspace groups. The DEX documents [say so](https://dexidp.io/docs/connectors/google/#fetching-groups-from-google). But the google docs say that DWD is not necessary (see [here](https://cloud.google.com/identity/docs/how-to/setup#auth-no-dwd)).

We are using DEX in terms of Argo CD and do not want to use domain wide delegation.

Thx!

**Comment by deepakdialpad** [comment]
Hello,
May i know where you able to get a solution for this ? We also require the same thing.
Thanks in advance.
